lib/tower_slack/reporter.ex

```diff
@@ -53,7 +53,7 @@ defmodule TowerSlack.Reporter do
          reason: reason,
          stacktrace: stacktrace
        }) do
-    post_message(id, similarity_id, "Exit", reason, stacktrace)
+    post_message(id, similarity_id, "Exit", Exception.format_exit(reason), stacktrace)
```

Comment:
Moving this to a more generalized PR https://github.com/mimiquate/tower_slack/pull/16.
This shouldn't be a Bandit specific change.